Library/OcFlexArrayLib/AsciiStringBuffer.c

```diff
@@ -27,19 +27,30 @@ OcAsciiStringBufferInit (
 
 EFI_STATUS
 OcAsciiStringBufferAppend (
-  IN OUT       OC_STRING_BUFFER    *Buffer,
-  IN     CONST CHAR8               *AppendString    OPTIONAL
+  IN OUT  OC_STRING_BUFFER    *Buffer,
+  IN      CONST CHAR8         *AppendString    OPTIONAL
   )
 {
   return OcAsciiStringBufferAppendN (Buffer, AppendString, MAX_UINTN);
 }
 
+/**
+  Extend string buffer by a specified length.
+
+  @param[in,out]  Buffer                String buffer to be extended.
+  @param[in]      AppendLength          Length to be appended.
+  @param[out]     TargetLength          Target length of the extended buffer.
+
+  @retval         EFI_SUCCESS           The buffer was extended successfully.
+  @retval         EFI_OUT_OF_RESOURCES  The extension failed due to the lack of resources.
+  @retval         EFI_UNSUPPORTED       The buffer has no size (This theoretically may never happen).
```

Comment:
This can happen on a release build, so I am not sure there is any use to the extra comment in brackets.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/f644d8672a5e59515ea8cab1dc660acbd6649ce2